agent/proxycfg/state.go

```diff
@@ -807,6 +807,7 @@ func (s *state) handleUpdateConnectProxy(u cache.UpdateEvent, snap *ConfigSnapsh
 				wildcardSID := structs.NewServiceID(structs.WildcardSpecifier, structs.WildcardEnterpriseMeta())
 				defaults, ok := snap.ConnectProxy.UpstreamConfig[wildcardSID.String()]
 				if ok {
+					u = defaults
```

Comment:
Is this already a value type or should you clone these defaults every time to prevent any incidental bleedover?

Reply:
`u` and `defaults` are pointers. What do you mean by bleedover? Modifying the upstream config for the defaults after we assign it to `u`?

Reply:
Yeah.